externals/coda-oss/modules/drivers/numpy/wscript

```diff
@@ -1,5 +1,6 @@
 import os, sys
 from os.path import join, exists
+from distutils import sysconfig
```

Comment:
Unused import?

Reply:
Resolved in coda-oss PR. I'll merge this and let the cleanup come in whenever it updates next.